x-pack/filebeat/input/cel/cel_metrics.go

```diff
@@ -0,0 +1,469 @@
+// +------------------------+-------------------------------------------------------+
+// | service.name,          | service that is running the program, ex. elastic-agent|
+// | deployment.environment | deployment environment, ex. production                |
+// +------------------------+-------------------------------------------------------+
+//
+// See [otel.ExportFactory] for environment settings to run console or http/protobuf output.
+// See [otelCELMetrics] for the complete list of exported metrics.
+
+// Open Telemetry Metrics for CEL Input
+//
+// CEL Metrics are sent as Delta metrics for each Periodic Run (Interval).
+// Export occurs as a push to an OTLP endpoint at the end of the Periodic Run. Metrics are reset between Periodic Runs.
+// +-------------------------------------------+----------------------------------------------------------------------------------------+------------------+
+// | name                                      | description                                                                            | metric type      |
+// |-------------------------------------------|----------------------------------------------------------------------------------------|------------------|
+// These metrics are generated by the OTEL SDK through wrapping the transport and are scoped in the OTEL metrics as
+// ' go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp'.
+// A metric is exported for each unique set of attributes:
+// - http.request.method
+// - http.response.status_code
+// - network.protocol.name
+// - network.protocol.version
+// - server.address
+// - server.port
+// - url.scheme
+//
+// +-------------------------------+----------------------------------------------+------------------+
+// | name                          | description                                  | metric type      |
+// |-------------------------------|----------------------------------------------|------------------|
+// | http.client.request.duration  | The duration in seconds for an http request  | Float64Histogram |
+// | http.client.request.body.size | The size of the request in bytes             | Int64Histogram   |
+// +------------------------------+-----------------------------------------------+------------------+
+//
+// See cel_metric_collection.png for a diagram of when each OTEL CEL metric is collected.
+//
+// inputMetrics
+//
+// inputMetrics are part of the agent monitoring framework. They are cumulative metrics for each input for the entire
+// run of a CEL input. They are packaged together with other agent data for monitoring.
+// Durations are collected as histograms and are in nanoseconds.
+// Cumulative HTTP metrics are collected through a wrapper on the HTTP transport.
+// See https://www.elastic.co/guide/en/beats/filebeat/current/http-endpoint.html
+// and https://www.elastic.co/docs/reference/fleet/monitor-elastic-agent for details
+// on agent monitoring.
```

Comment:
This big comment block at the top of `cel_metrics.go` won’t show up in godoc because it’s not the package comment (and we can only have one package-level comment; currently that’s in `input.go`).

The most appropriate place for this content is a `doc.go` file in `x-pack/filebeat/input/cel` with a `// Package cel ...` comment. Move/merge the existing package comment from `input.go` plus the OTEL/resource attributes/metrics documentation into a package level `doc.go` comment.